assets/js/frontend-components/price-slider/index.js

```diff
@@ -57,50 +50,17 @@ class PriceSlider extends Component {
 		if ( '' === value ) {
 			return '';
 		}
+		const { currencySymbol, priceFormat } = this.props;
+
 		const formattedNumber = parseInt( value, 10 );
-		const currencySymbol = get( wcSettings, [ 'currency', 'symbol' ], '$' );
-		const priceFormat = get( wcSettings, [ 'currency', 'price_format' ], '%1$s%2$s' );
+		const formattedValue = sprintf( priceFormat, currencySymbol, formattedNumber );
```

Comment:
What about currencies that have the symbol _after_ the number and not before?  With that in mind, I wonder if we should make a currency value object that we can pass around (and more easily type check for) as a prop?  I built a [currency vo for Event Espresso](https://github.com/eventespresso/event-espresso-core/blob/master/assets/src/vo/currency.js) - we could do something similar?

Reply:
Price format handles this :)

Reply:
ahh, meaning it could be `%2$s%1$s`?

Reply:
Yup thats it. It comes from the PHP side which already has this system in place.